junit-platform-suite-commons/src/main/java/org/junit/platform/suite/commons/SuiteLauncherDiscoveryRequestBuilder.java

```diff
@@ -65,7 +65,46 @@
 import org.junit.platform.suite.api.SelectUris;
 
 /**
+ * The {@code SuiteLauncherDiscoveryRequestBuilder} provides a builder for constructing instances
+ * of {@link LauncherDiscoveryRequest} specifically tailored for suite execution.
+ *
+ * <h2>Example</h2>
+ *
+ * <pre>{@code
+ * SuiteLauncherDiscoveryRequestBuilder.request()
+ * 		.selectors(
```

Comment:
For future reference, please note that we use spaces in Javadoc instead of tabs.

I'm addressing that locally, so there's no need to update the PR.